src/ActiveLogin.Authentication.BankId.AspNetCore/Qr/BankIdDefaultQrCodeGenerator.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.IO;
+using System.Reflection;
+using Microsoft.Extensions.Localization;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Qr
+{
+    internal class BankIdDefaultQrCodeGenerator : IBankIdQrCodeGenerator
```

Comment:
`BankIdMissingQrCodeGenerator` is probably a better name

Reply:
👍